ci/jenkins/Test.groovy.j2

```diff
@@ -183,7 +183,11 @@
   ci_setup(ci_arm)
   sh (
     script: "${docker_run} ${ci_arm} ./tests/scripts/task_python_frontend_cpu.sh",
-    label: 'Run Python frontend tests',
+    label: 'Run General Python frontend tests',
+  )
+  sh (
```

Comment:
Could you push this PR branch to apache/tvm as well? That will kick off a CI job `tvm-ci/branch` that will test these Jenkinsfile changes

Reply:
Sure will close this one and submit a new PR